src/acl/InnerNode.cc

```diff
@@ -82,7 +82,7 @@ Acl::InnerNode::dump() const
 {
     SBufList rv;
     for (Nodes::const_iterator i = nodes.begin(); i != nodes.end(); ++i)
-        rv.push_back(SBuf((*i)->name));
+        rv.push_back((*i)->name);
```

Comment:
AFAIK this should be a ranged-for now, which simplifies this syntax quite a lot. Please consider upgrading this fully.

Reply:
Done to avoid arguing (commit 216013e).